src/components/AddEditCompany.js

```diff
@@ -2,21 +2,24 @@ import React, { Component } from 'react'
 import { db } from '../config/constants'
 import EditForm from './EditForm.jsx'
 import firebase from 'firebase'
-require('firebase/firestore');
+require('firebase/firestore')
 
 export default class AddEditCompany extends Component {
     constructor(props) {
         super(props)
         this.state = {
-            companyName: '',
+            dynamicCompanyName: '',
```

Comment:
Added dynamic and static company names so we can change names of companies in the future. It was also needed because companyName was a site effect in updateCompanyData